### DISCUSSION TITLE
How to Remove  Custom Site from Scrapers

### DISCUSSION THREAD

**Discussion by Conkuist** [xbapps/xbvr#1843] [Q&A]
I used the "Advanced / Create Custom Site" option to add a new site.
Is there a way to remove the site I created from the list of scrapers?
Not sure if it is relevant or not but I'm using XBVR with MariaDB.

**Comment by eromagosa** [comment]
+1

**Reply by NoEscape** [reply]
I have the same question. I suppose you could delete the row out of the sites table but looking at the schema and browsing the table I don't see the scraper URL in any of the columns.

Edit: ok, it's in the scraper.json file. Maybe deleting it from the "custom" section will do the trick.
